varia/sass/layout/_structure.scss

```diff
@@ -151,8 +151,17 @@
 	}
 
 	/* Reset alignleft and alignright margins after alignfull */
-	&.alignfull + .alignleft,
+	&.alignfull + .alignleft {
+		float: left;
+		margin-top: map-deep-get($config-global, "spacing", "vertical");
+		margin-left: 1rem;
```

Comment:
Let’s use the horizontal spacing variable(`#{map-deep-get($config-global, "spacing", "horizontal")};`) for `margin-left` and `margin-right`.